warehouse/macros/get_latest_schedule_data.sql

```diff
@@ -0,0 +1,26 @@
+{% macro get_latest_schedule_data(latest_only_source, table_name, clean_table_name) %}
```

Comment:
this is mostly a nitpick but usually macros produce a full query that you wrap in a subquery or a CTE; so the call ends up looking like

```
with latest as ( {{ get_latest_schedule_data(...) }} )

select * from latest
```

Reply:
Resolved 19387b1dd54c68c7e20e573f2b6cd7150b1e18e0